tools/migrations/2022_07-19_noNullUserIdReviewNew.js

```diff
@@ -0,0 +1,13 @@
+export const up = async ({ Sequelize, sequelize }) => {
```

Comment:
Sorry, sorry, but I have to tell you that this file is named inconsistently with all the rest of them, using a dash instead of an underscore!

Reply:
lol good catch